kotlin-native/Interop/Indexer/prebuilt/nativeInteropStubs/c/clangstubs.c

```diff
@@ -68,7 +68,7 @@ JNIEXPORT void JNICALL Java_clang_clang_kniBridge19 (JNIEnv* jniEnv, jclass jcls
 }
 JNIEXPORT void JNICALL Java_clang_clang_kniBridge20 (JNIEnv* jniEnv, jclass jclss, jlong p0, jlong p1) {
     CXString kniStructResult = clang_getFileName((void*)p0);
-    memcpy(p1, &kniStructResult, sizeof(kniStructResult));
+    memcpy((void*)p1, &kniStructResult, sizeof(kniStructResult));
```

Comment:
This file is auto-generated (see in the beginning of the file). Therefore, patching it manually is undesirable.
See the task used for the generation: https://github.com/JetBrains/kotlin/blob/1dad8906521528e0a63fe89f4c3f98ed42957a52/kotlin-native/Interop/Indexer/build.gradle.kts#L200